docs/specs/test-specs.md

```diff
@@ -0,0 +1,39 @@
+# Dusty Testing Specs
+These specs are used to specify tests you can run in apps and libs. These do not exist as distinct documents but as subfields of lib and app specs. They are found under the **test** key.
+
+## Keys
+
+### build and image
+The image and build keys in test specs mirror the image and build keys in the [app-spec](./app-specs.md#build).
```

Comment:
do some noting of that with the once command and the services command. Let me know if you think i need more or should consolidate them somewhere.